draft-ietf-teep-architecture.md

```diff
@@ -1347,6 +1347,33 @@ still a threat present in allowing a potentially compromised TAM to install
 any TA it wants; for example, a known-compromised TA or an explicitly
 malicious TA.
 
+The device manufacturer may want to include a notification support that can allow
+Device Administrators be notified for incidents upon detection with some mitigation 
+methods as suggested above.
+
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs where a TAM turns to act out of 
```

Comment:
```suggestion
There are also threats of hostile or abusive TAMs where a TAM is compromised and thus turns to act out of 
```
Is this what you mean by "turns"?  If instead you mean it's intentional by the TAM but not by the Device Administrator who chose to trust that TAM then

```suggestion
There are also threats of hostile or abusive TAMs where a TAM acts out of 
```